.gitignore

```diff
@@ -3,3 +3,6 @@
 local.properties
 build
 .gradle
+.project
```

Comment:
I would add the following comment as well, "# Eclipse project files" also probably include ".classpath" as well